### PR TITLE
[ONNX] Extend se block ignored pattern

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Add coverage of SE block in torchvision mobilenet_v3 for ONNX.
image

### Reason for changes

Improve accuracy

### Related tickets

150204

### Tests

Manually on the provided model.

### CODE REVIEW

**Review by daniil-lyakhov — APPROVED** [review]

**Comment by kshpv** [comment]
There is no impact on timm model from PTQ scope

File: nncf/onnx/quantization/ignored_patterns.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def create_se_block() -> GraphPattern:
     reduce_mean_node = pattern.add_node(
         **{
             GraphPattern.LABEL_ATTR: "REDUCE_MEAN",
-            GraphPattern.METATYPE_ATTR: om.ONNXReduceMeanMetatype,
+            GraphPattern.METATYPE_ATTR: [om.ONNXReduceMeanMetatype, om.ONNXGlobalAveragePoolMetatype],
             GraphPattern.PATTERN_NODE_TO_EXCLUDE: True,
         }
```
[the remark this answers is not in the document]
Comment (kshpv) [reply nncf/onnx/quantization/ignored_patterns.py]: done

[the remark this answers is not in the document]
Comment (KodiaqQ) [reply nncf/onnx/quantization/ignored_patterns.py]: Is that a mistake? Or a joke?
It was renamed as `POOLING` for `[om.ONNXReduceMeanMetatype, om.ONNXGlobalAveragePoolMetatype]`.
Even the `reduce_mean_node` variable remains the same.

[the remark this answers is not in the document]
Comment (kshpv) [reply nncf/onnx/quantization/ignored_patterns.py]: I am very sorry for the incorrect changes. Is `reduce_node_or_global_average_pool` for the pattern node and `"REDUCE_MEAN || GLOBAL_AVERAGE_POOL"` for LABEL_ATTR  in your mind?
Thank you!

[the remark this answers is not in the document]
Comment (KodiaqQ) [reply nncf/onnx/quantization/ignored_patterns.py]: Yes.

[the remark this answers is not in the document]
Comment (kshpv) [reply nncf/onnx/quantization/ignored_patterns.py]: updated

**Review by KodiaqQ — APPROVED** [review]

**Review by daniil-lyakhov — APPROVED** [review]

**Pull request merged**